src/systems/ec/implementations/react/packages/file/FileDownload.jsx

```diff
@@ -0,0 +1,147 @@
+  }
+
+  render() {
+    const {
+      title,
+      language,
+      meta,
+      icon,
+      download,
+      translation,
+      className,
+      ...props
+    } = this.props;
+
+    const classNames = classnames(className, 'ecl-file');
+
+    return (
+      <div {...props} className={classNames} ref={this.fileDownloadRef}>
+        <div className="ecl-file__container">
+          <Icon {...icon} className={classnames(className, 'ecl-file__icon')} />
+
+          <div className="ecl-file__info">
+            <div className="ecl-file__title">{title}</div>
+            <div className="ecl-file__language">{language}</div>
+            <div className="ecl-file__meta">{meta}</div>
+          </div>
+
+          <Link
+            {...download}
+            variant="standalone"
+            className={classnames(
+              className,
+              'ecl-file__download',
+              'ecl-button',
+              'ecl-button--secondary'
+            )}
+          />
+        </div>
+
+        {!!(
+          translation &&
+          translation.items &&
+          translation.items.length > 0
+        ) && (
+          <div
+            className="ecl-file__translation-container"
+            data-ecl-file-translation-container
+          >
+            <Link
+              {...translation.toggle}
+              variant="standalone"
+              className={classnames(className, 'ecl-file__translation-toggle')}
+              data-ecl-file-translation-toggle
+            />
+
+            <ul className="ecl-file__translation-list">
+              {translation.items.map(item => (
+                <li
+                  className={classnames(
+                    item.className,
+                    'ecl-file__translation-item'
+                  )}
+                  key={item.title}
+                >
+                  <div className="ecl-file__translation-info">
+                    <div className="ecl-file__translation-title">
+                      {item.title}
+                    </div>
+
+                    <div className="ecl-file__translation-meta">
+                      {item.meta}
+                    </div>
+                  </div>
+
+                  <Link
+                    {...download}
+                    variant="standalone"
+                    className={classnames(
+                      className,
+                      'ecl-file__translation-download'
+                    )}
+                  />
+                </li>
+              ))}
+
+              <li className="ecl-file__translation-item ecl-file__translation-description">
+                {translation.description}
+              </li>
+            </ul>
+          </div>
+        )}
+      </div>
+    );
+  }
+}
+
+FileDownload.propTypes = {
+  title: PropTypes.string,
+  language: PropTypes.string,
+  meta: PropTypes.string,
+  download: PropTypes.shape(Link.propTypes),
+  translation: PropTypes.shape({
```

Comment:
```
Warning: Failed prop type: Invalid prop `translation` of type `array` supplied to `FileDownload`, expected `object`.
```
thrown in storybook because of the default value `translation: [],`